File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cs
public class Solution {
      public string MergeAlternately(string word1, string word2)
      {
            int Mxsize = Math.Max(word1.Length, word2.Length);
            string ans = "";
            for (int i = 0; i <Mxsize ; i++)
            {
                  if (i < word1.Length) ans += word1[i];
                  if (i < word2.Length) ans += word2[i];
                  
            }
            
            
            return ans;
      }
}

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int Mxsize =max(word1.size(), word2.size());
            string ans = "";
            for (int i = 0; i <Mxsize ; i++)
            {
                  if (i < word1.size()) ans += word1[i];
                  if (i < word2.size()) ans += word2[i];
                  
            }
            
            
            return ans;
    }
};